test/samples/array-expressions/expected.js

```diff
@@ -1,9 +1,9 @@
 a = [1, 2, 3];
 
 b = [
-	"the quick brown fox jumps over the lazy dog",
-	"the quick brown fox jumps over the lazy dog",
-	"the quick brown fox jumps over the lazy dog",
+	'the quick brown fox jumps over the lazy dog',
```

Comment:
the quotation change is because that `node.raw` itself is wrapped with single quotes rather than double quotes?

this might break all snapshot tests downstream right?

Reply:
`node.raw` is whatever string was used in the original parsed source for the particular literal. So these are single quotes because they were single quotes in `input.js`. I think this seems mildly nicer than always using double quotes and `JSON.stringify()`. Yeah this would break a lot of downstream js snapshot tests, but I don't really see that as a big problem. This library doesn't make any semver guarantees about what equivalent strings it uses for what expressions, and the snapshot tests are expected to be brittle and are easy to update.